test/images/image-util.sh

```diff
@@ -115,41 +115,41 @@ docker_version_check() {
 # This function will push the docker images
 push() {
   docker_version_check
-  TAG=$(<${IMAGE}/VERSION)
+  TAG=$(<"${IMAGE}"/VERSION)
   if [[ -f ${IMAGE}/BASEIMAGE ]]; then
     archs=$(listArchs)
   else
     archs=${!QEMUARCHS[@]}
   fi
   for arch in ${archs}; do
-    docker push ${REGISTRY}/${IMAGE}-${arch}:${TAG}
+    docker push "${REGISTRY}/${IMAGE}-${arch}:${TAG}"
   done
 
   kube::util::ensure-gnu-sed
 
   # The manifest command is still experimental as of Docker 18.09.2
   export DOCKER_CLI_EXPERIMENTAL="enabled"
   # Make archs list into image manifest. Eg: 'amd64 ppc64le' to '${REGISTRY}/${IMAGE}-amd64:${TAG} ${REGISTRY}/${IMAGE}-ppc64le:${TAG}'
-  manifest=$(echo $archs | ${SED} -e "s~[^ ]*~$REGISTRY\/$IMAGE\-&:$TAG~g")
-  docker manifest create --amend ${REGISTRY}/${IMAGE}:${TAG} ${manifest}
+  manifest=$(echo "$archs" | ${SED} -e "s~[^ ]*~$REGISTRY\/$IMAGE\-&:$TAG~g")
+  docker manifest create --amend "${REGISTRY}/${IMAGE}:${TAG}" "${manifest}"
```

Comment:
This line is broken. To reproduce, run `make push-all WHAT=webhook`, it returns error message "invalid referent format".

Reply:
@caesarxuchao Thanks for your feedback.  I'll check it again.

Reply:
Thanks for catching this @caesarxuchao

Reply:
I'm sorry for missing it. Thank you @caesarxuchao @listx